src/components/autocomplete/autocomplete.scss

```diff
@@ -0,0 +1,161 @@
+@keyframes list-out {
+  0% {
+    animation-timing-function: linear;
+  }
+  50% {
+    opacity: 0;
+    height: 40px;
+    animation-timing-function: ease-in;
+  }
+  100% {
+    height: 0;
+    opacity: 0;
+  }
+}
+@keyframes list-in {
+  0% {
+    opacity: 0;
+    height: 0;
+    animation-timing-function: linear;
+  }
+  50% {
+    opacity: 1;
+    height: 40px;
+    animation-timing-function: ease-out;
+  }
+  100% {
```

Comment:
(Sorry if it is obvious, but) what is the purpose of these keyframes having identical values for 50% and 100% ?